src/Calculator/Calculator.vcxproj

```diff
@@ -390,6 +391,7 @@
     <ClCompile Include="Converters\ItemSizeToVisibilityConverter.cpp" />
     <ClCompile Include="Converters\RadixToStringConverter.cpp" />
     <ClCompile Include="Converters\VisibilityNegationConverter.cpp" />
+    <ClCompile Include="Controls\HorizontalNoOverflowStackPanel.cpp" />
```

Comment:
I'm a little confused, what's the purpose of this filter when it already matches the real folder structure?

Reply:
It will help VisualStudio organize the files into the Controls folder when viewing from the solutions view instead of the files view. To be honest, I am not sure why this isn't done automatically, but you can make this change by just dragging the new files into the controls folder from VS.

![image](https://user-images.githubusercontent.com/6334170/54646529-8debd380-4a5c-11e9-9df4-ec5ea1580424.png)

Reply:
TIL the Solution Explorer use Filters but not folder structure when `Show All Files` is off (but use the Folder structure when it's on), weird behavior! I will change that!